contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java

```diff
@@ -45,46 +47,36 @@
  * @since 2.1.0
  */
 @Evolving
-public class CassandraPOJOOutputOperator extends AbstractCassandraTransactionableOutputOperatorPS<Object>
+public class CassandraPOJOOutputOperator extends AbstractCassandraTransactionableOutputOperatorPS<Object> implements Operator.ActivationListener<Context.OperatorContext>
 {
-  private static final long serialVersionUID = 201506181024L;
   @NotNull
-  private ArrayList<String> columns;
-  private final transient ArrayList<DataType> columnDataTypes;
+  private List<FieldInfo> fieldInfos;
   @NotNull
-  private ArrayList<String> expressions;
-  private final transient ArrayList<Object> getters;
-
-  /*
-   * An ArrayList of Java expressions that will yield the field value from the POJO.
-   * Each expression corresponds to one column in the Cassandra table.
-   */
-  public ArrayList<String> getExpressions()
-  {
-    return expressions;
-  }
+  private String tablename;
 
-  public void setExpressions(ArrayList<String> expressions)
-  {
-    this.expressions = expressions;
-  }
+  protected final transient ArrayList<DataType> columnDataTypes;
+  protected final transient ArrayList<Object> getters;
+  protected transient Class<?> pojoClass;
 
-  /*
-   * An ArrayList of Columns in the Cassandra Table.
+  /**
+   * The input port on which tuples are received for writing.
    */
-  public ArrayList<String> getColumns()
+  @InputPortFieldAnnotation(optional = true)
```

Comment:
Is this input port intended to be optional? This hides the original field in superclass.